gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.3-all.zip
```

Comment:
`bin` is good enough

Reply:
Technically, but it's a pain in the butt when you're doing Gradle things and want to click into the source or even just read the javadoc.

Since Gradle's not being bundled into the release, it doesn't do any harm to just improve the devx a little bit.